packages/ubuntu_desktop_installer/lib/utils/version_extractor.dart

```diff
@@ -0,0 +1,65 @@
+import 'dart:io';
+
+class VersionExtractor {
```

Comment:
The name of the class isn't really self-explanatory (it extracts more than just a version number). What about `ProductInfoExtractor`? Also, it could really use documentation to explain what it does and for its public members.